Modules/Forum/classes/class.ilForumSettingsGUI.php

```diff
@@ -414,14 +425,62 @@ private function showMembersTable(array $moderators, array $admins, array $membe
 
     public function saveEventsForUser()
     {
-        $hidden_value = json_decode($_POST['hidden_value']);
+        /** @var $DIC  \ILIAS\DI\Container */
+        global $DIC;
```

Comment:
Please try to move the `global $DIC` to the constructor and assign it to a class property. Then, access the class property here.